.github/workflows/deploy.yml

```
@@ -73,7 +73,7 @@ jobs:
           - os: windows-2019
             target: win
             node: 20
-          - os: macos-11
+          - os: macos-14-large
```

We should use the lowest available option: https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md

```suggestion
          - os: macos-12
```